### ISSUE TITLE
Why the recogntion accruacy of SVM is much lower than the fine_tuning test accuracy ?

### ISSUE THREAD

**Issue by tzczsq** [facebook/C3D#109]
Hi, I have successfully fine-tuned your trained model on own datasets.
As described below, the test score is 0.5485 with iteration 1400.
However, following in your work, when I average the video clip into a 4096-dim features, we only obtain an accuracy of about 19%  with a linear SVM classifier. 
Why not the test accuracy with the average features  is 19%, rather than 54.85%?
I am very confused with the different results.

%%%%%%%%%%%%%%%%%%%%
I0417 13:48:44.167778 28907 solver.cpp:106] Iteration 1400, Testing net
I0417 13:49:34.865902 28907 solver.cpp:142] Test score #0: 0.5485
I0417 13:49:34.865955 28907 solver.cpp:142] Test score #1: 1.15081
I0417 13:50:11.431813 28907 solver.cpp:237] Iteration 1420, lr = 1e-05
I0417 13:50:11.434947 28907 solver.cpp:87] Iteration 1420, loss = 1.51454
I0417 13:50:49.284466 28907 solver.cpp:237] Iteration 1440, lr = 1e-05
I0417 13:50:49.287461 28907 solver.cpp:87] Iteration 1440, loss = 1.17923
I0417 13:51:26.970572 28907 solver.cpp:237] Iteration 1460, lr = 1e-05
I0417 13:51:26.973203 28907 solver.cpp:87] Iteration 1460, loss = 0.964503
I0417 13:52:04.917403 28907 solver.cpp:237] Iteration 1480, lr = 1e-05
I0417 13:52:04.920508 28907 solver.cpp:87] Iteration 1480, loss = 1.71238
I0417 13:52:42.893206 28907 solver.cpp:237] Iteration 1500, lr = 1e-05
I0417 13:52:42.895956 28907 solver.cpp:87] Iteration 1500, loss = 1.1381
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

**Comment by dutran** [comment]
If your dataset is significantly different from Sports1M, so finetuning in that case should work much better because it is non-linear model and feature maps are adapted to this new dataset. On the other hand, fix fc6 feature (with was trained on Sports1M), then use linear SVM (linear model) on your dataset may give a lower accuracy.

I can expect the lower performance compared to finetuned net, but the gap is quite high to me though.
